lib/raster/put_title.c

```diff
@@ -66,6 +70,7 @@ int Rast_put_cell_title(const char *name, const char *title)
         fclose(in);
         G_warning(_("can't write category information for [%s] in [%s]"), name,
                   mapset);
+        G_free(tempfile);
```

Comment:
Here tempfile may be removed, (`in`) is successfully assigned.

Reply:
done